spec/requests/bulk_export_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+
+describe BulkExportController do
+  let!(:admin) { create(:admin) }
+  let!(:regular_user) { create(:user) }
+  let!(:collection) { create(:collection, owner_user_id: admin.id) }
```

Comment:
We need to create an owner user and that owner user should own the collection, rather than the admin.